services/collaboration/pkg/connector/fileconnector.go

```diff
@@ -798,7 +854,8 @@ func (f *FileConnector) DeleteFile(ctx context.Context, lockID string) (*Connect
 		if deleteRes.GetStatus().GetCode() == rpcv1beta1.Code_CODE_TOO_EARLY {
 			// starting from 20ms, double the waiting time for each retry
 			// capping at 5 secs
-			waitingTime := (20 * time.Millisecond) << retries
+			var waitingTime time.Duration
```

Comment:
not sure if this improves anything or it's just an extra line we can remove.

Reply:
my IDE didn't understand the type of `waitingTime`